content/support-matrix/delfin/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Delfin
+menuTitle: Delfin
+description: "All support matrix data for drivers of Delfin project"
+weight: 10
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"] 
+---
+
+All support matrix data documents of Delfin drivers can be found here.
```

Comment:
Will any documents will be added? If not then just data or tables.

Reply:
Changed the statement